c2corg_ui/templates/helpers/view.html

```diff
@@ -97,15 +97,22 @@
 
 <%def name="show_associated_routes(document)">\
 
-    % for r in document['associations']['routes']:
+    % for route in document['associations']['routes']:
 
     <div class="list-item">
-      % if 'title_prefix' in r['locales'][0] and r.locales[0]['title_prefix']:
-        <a href="${request.route_url('routes_view', id=r['document_id'], lang=r['locales'][0]['lang'])}">${r['locales'][0]['title']}</a>
+      % if 'title_prefix' in route['locales'][0] and route.locales[0]['title_prefix']:
+        <a href="${request.route_url('routes_view', id=route['document_id'], lang=route['locales'][0]['lang'])}">${route['locales'][0]['title']}</a>
       % else :
-        <a href="${request.route_url('routes_view', id=r['document_id'], lang=r['locales'][0]['lang'])}">${r['locales'][0]['title']}</a>
+        <a class="item-title" href="${request.route_url('routes_view', id=route['document_id'], lang=route['locales'][0]['lang'])}">${route['locales'][0]['title']}
+          <div class="route-activities">
+            ${get_route_activities(route, 'top')}
+          </div>
+          <div>
+            <p><b translate>elevation</b>  min ${route['elevation_min']} / max ${route['elevation_max']}</p>
```

Comment:
Note that `elevation_min/max` are not required fields, so you might end up with: "elevation min / max".